### DISCUSSION TITLE
Grain boundary plane distribution

### DISCUSSION THREAD

**Discussion by mugikumar** [mtex-toolbox/mtex#2055] [Ask Anything]
Hello everyone 
       I analysed the grain boundary plane distribution with a specific misorientation angle.  I want to know whether I have done the correct thing.
gB = grains.boundary('indexed');
cond = gB.misorientation.angle > 47 * degree;
![50gbt_05](https://github.com/mtex-toolbox/mtex/assets/109027461/1769e0be-483b-4a4f-8b2b-130cb5ec1e6a)
In this result, the grain boundaries are more parallel to the 110 plane. Is this correct

**Comment by kilir** [comment]
Hi,
I think you need to supply a little more information on what you do.
Cheers,

Rüdiger

**Comment by mugikumar** [comment]
Hi Kilir,
          I have to analyze the plane distribution on specific misorientation angle boundaries. I read one piece of literature they have done on plane distribution with irrespective of misorientation angle. Please check it down. Similarly, I want to analyse the 50- to 60-degree misorientation with plane distribution on their boundaries. 
![image](https://github.com/mtex-toolbox/mtex/assets/109027461/35003ff6-c9d3-411a-9e50-1c10c65738b1)
I think so, and this is possible on the grain boundary plane distribution method. Am I correct

**Comment by kilir** [comment]
I think you still forgot to give the relevant lines of your script which finally produce the plot.

But let's say you used `gB(cond)` as an input of  `calcGBPD()` and your boundaries are sufficiently smooth, this should be fine.

Cheers,
Rüdiger

**Comment by mugikumar** [comment]
Hi Kilir,
            Thanks for your reply. My question is: there are two kinds of misorientation angles. They are overall misorientation angles and specific misorientation angles. OMA means the lowest rotation angle with adjacent crystal, and SMA means the minimum misorientation angle of a crystallographic plane. Are SMA and GBPD the same? If not, is it possible to get SMA by Mtex
